src/web/CustomScrollbar.ts

```diff
@@ -329,7 +329,24 @@ export class Scrollbar {
     }
 
     private _handleMouseDown(e: React.MouseEvent<any>) {
-        this._prevent(e);
+        const target = e.currentTarget as any;
+
+        if (this._dragging === true || e.target === this._verticalBar.slider || !target) {
+            this._prevent(e);
+            return;
+        }
+
+        if (this._hasVertical) {
+            const eventOffsetY = e.pageY - target.getBoundingClientRect().top;
+            const halfHeight = this._verticalBar.slider.offsetHeight / 2; 
+            const offsetY = (eventOffsetY - this._verticalBar.slider.offsetTop - halfHeight) * this._verticalBar.slider2Scroll;
+            this._viewport.scrollTop = offsetY + this._viewport.scrollTop;
+        } else if (this._hasHorizontal) {
+            const eventOffsetX = e.pageX - target.getBoundingClientRect().left;
+            const halfWidth = this._horizontalBar.slider.offsetWidth / 2; 
+            const offsetX = (eventOffsetX - this._horizontalBar.slider.offsetLeft - halfWidth) * this._verticalBar.slider2Scroll;
```

Comment:
This should use this._horizontalBar, not this._verticalBar.

Reply:
argh old version